core/modules/widgets/navigator.js

```diff
@@ -549,13 +549,25 @@ NavigatorWidget.prototype.handlePerformImportEvent = function(event) {
 	var self = this,
 		importTiddler = this.wiki.getTiddler(event.param),
 		importData = this.wiki.getTiddlerDataCached(event.param,{tiddlers: {}}),
-		importReport = [];
+		importReport = [],
+	    	moreTags = this.wiki.getTiddlerText("$:/language/DefaultNewTiddlerTags");
 	// Add the tiddlers to the store
 	importReport.push($tw.language.getString("Import/Imported/Hint") + "\n");
 	$tw.utils.each(importData.tiddlers,function(tiddlerFields) {
 		var title = tiddlerFields.title;
 		if(title && importTiddler && importTiddler.fields["selection-" + title] !== "unchecked") {
-			var tiddler = new $tw.Tiddler(tiddlerFields);
+			var tiddler;
+			if(moreTags && tiddlerFields.tags) {
+				tiddler = new $tw.Tiddler(tiddlerFields,{
+					tags: tiddlerFields.tags + " " + moreTags
+				});
+			} else if (moreTags) {
+				tiddler = new $tw.Tiddler(tiddlerFields,{
+					tags: moreTags
+				});
+			} else {
+				tiddler = new $tw.Tiddler(tiddlerFields);
+			}
 			tiddler = $tw.hooks.invokeHook("th-importing-tiddler",tiddler);
```

Comment:
Your "old" code was inserted just above this line. So I think it would be a perfect fit for a hook - plugin.